src/Altinn.Platform/Altinn.Platform.Events/Functions/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# Azure Functions localsettings file
```

Comment:
Isn't this project covered by the general .gitignore in the repo?

Reply:
Auto-generated when creating new project. Removed